docs/source/additional/privacy-strategy.rst

```diff
@@ -0,0 +1,133 @@
+
+.. note::
+
+    We are aware that our initial assumption may seem restrictive, but we make this assumption because Substra does not provide protection against malicious actors within a closed network. The trust is here ensured through non-technical means - the organizations are honest due to liabilities, regulations and contracts. This excludes any wide open federated networks, where data is made available to any public researcher.
+
+Following these assumptions, the privacy threats when performing Federated Learning can be classified in two categories. 
+
+#. **Generic cyber-security attacks**: 
+if a malicious actor can get access to the internal infrastructure, they can exfiltrate some sensitive data (or cause other kinds of mayhem). This is not specific to FL settings, but the inherent decentralization of FL does reduce the severity of such breaches despite the fact that each communication channel with the external world is a potential attack surface, and by design, part of the code is executed on remote machines.
+#. **Attacks specific to FL**:
+These are attacks related to the information contained in the mathematical objects exchanged when training a model. Said otherwise, the model updates and/or the final trained model parameters might encode sensitive information about the training dataset. These may be relevant for pure machine learning as well but are exacerbated in FL as the data is often viewed by a model many times. Examples of such threats include:
+   
+   #. **Membership attacks**: 
+   When a final trained model is used to try to guess whether a specific data sample was used during training [Membership Inference Attacks against Machine Learning Models, Shokir et al. 2016]. Membership attack is not specific to FL, as it relies on the final trained model. It can be performed in the two following settings:
+      
+      #. **Black box attack**:
+      This is an attack made from the prediction of a trained model on a given set of samples. Black box attack is an attack which requires the minimal amount of rights/permissions from the attacker. For example, only an API to request model prediction is provided to the attacker.
+      #. **White box attack**:  
+      An attack where the attacker needs to access the architecture and weights of a trained model
+  
+   #. **Reconstruction attacks**: 
+   When the batch gradient or the FL model updates are used to reconstruct from scratch a data sample used during the training. [Inverting Gradients - How easy is it to break privacy in federated learning?, Geiping et al. 2020]. 
```

Comment:
Do we want to provide a link to the paper? Else, we should at least format the reference